src/formatDate.js

```diff
@@ -8,7 +8,33 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const splitted = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
The code assumes that the last element of `fromFormat` is the delimiter. This might not always be the case, as the delimiter could be different or not present as the last element. Consider revising the logic to handle delimiters more flexibly.